[14.0][ADD] edi_sale_endpoint

thienvh332 opened the pull request:
Depends on:
- PR #23

simahawk committed:
Add edi_sale_oca

simahawk committed:
edi_sale: take over edi_sale_order_import

simahawk committed:
edi_sale: filter/group by origin

simahawk committed:
edi_sale: handle ext ref for sol

simahawk committed:
edi_sale: show edi_id

simahawk committed:
edi_sale: store metadata

simahawk committed:
edi_sale: clean file and drop *type_auto dependency

simahawk committed:
edi_sale: add process hook for order create

simahawk committed:
edi_sale: process propagate edi origin

simahawk committed:
edi_sale: add edi_exchange_ready

simahawk committed:
edi_sale: fix edi disable flag

simahawk committed:
Add edi_sale_endpoint

simahawk committed:
edi_sale: fix _edi_exchange_ready for qty=0

Lines w/ no qty must not be excluded a priori

simahawk committed:
edi_sale: move OrderMixin from edi_sale_ubl

simahawk committed:
edi_sale: fix test_process

simahawk committed:
edi_sale: propagate origin on lines

Commit:
[MIG] edi_sale_oca: Migration to 16.0

Commit:
[IMP] edi_sale_oca: Support edi_id for edi_sale_edifact_oca

Commit:
[IMP] edi_sale_oca: Refactor code for passing all configurations via env_ctx

The configurations here are values for the wizard and whatever we need

Commit:
[UPD] Update edi_sale_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[IMP] edi_oca: add partner form page

Commit:
[UPD] Update edi_sale_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[FIX] edi_sale_oca: fix test

Commit:
[UPD] Update edi_sale_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update edi_sale_oca.pot

mymage committed:
Added translation using Weblate (Italian)

thienvh332 committed:
[BPRT] edi_sale_oca: Backport from v16.0

thienvh332 committed:
[IMP] edi_sale_oca: add partner form page

QuocDuong1306 reviewed (approved):
pre-approving

maybe missing `CREDITS.rst`, no? and also we need to squash all fixup commits

alexis-via committed:
[FIX] account_invoice_import_simple_pdf: version extraction for fitz/PyMuPDF

Use with statement for NamedTemporaryFile()

alexis-via committed:
[FIX] account_invoice_import_simple_pdf: add seek(0) to avoid error "empty file"

Add try/except on pypdf text extraction

thienvh332 committed:
[FIX] pdf_helper: fix issue pypdf 5.0 dropped support for python<=3.7

alexis-via committed:
[FIX] account_invoice_download: don't set default_name in context

If default_name is set in context just to have a default label on the
account.invoice.import.download, it will also be used if do a manual run
right after, so you will get draft invoices with a 'name' already set !

OCA-git-bot committed:
Merge PR #1041 into 14.0

Signed-off-by simahawk

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1018 into 14.0

Signed-off-by alexis-via

OCA-git-bot committed:
[BOT] post-merge updates

alexis-via committed:
[FIX] pypdf version for tests

OCA-git-bot committed:
Merge PR #1043 into 14.0

Signed-off-by alexis-via

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1023 into 14.0

Signed-off-by simahawk

Commit:
[UPD] Update edi_oca.pot

Commit:
[UPD] Update edi_sale_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: edi-14.0/edi-14.0-edi_oca
Translate-URL: https://translation.odoo-community.org/projects/edi-14-0/edi-14-0-edi_oca/
